code/controllers/subsystem/adjacent_air.dm

```diff
@@ -0,0 +1,35 @@
+SUBSYSTEM_DEF(adjacent_air)
+	name = "Atmos Adjacency"
+	flags = SS_BACKGROUND
+	runlevels = RUNLEVEL_GAME | RUNLEVEL_POSTGAME
+	wait = 10
```

Comment:
Any further discussion for this number?
https://github.com/tgstation/tgstation/pull/40725#discussion_r222858852